aws-mssql.yml

```diff
@@ -11,8 +11,31 @@ support_url: https://aws.amazon.com/sql/
 tags: [aws, mssql, beta]
 plan_updateable: true
 provision:
-  plan_inputs: [] # Expected: mssql_version, storage_gb
+  plan_inputs: []
   user_inputs:
+  - field_name: engine
+    required: true
+    type: string
+    details: The edition for the MSSQL instance.
+    default: sqlserver-ee
+    enum:
+      sqlserver-ee: sqlserver-ee
+      sqlserver-ex: sqlserver-ex
+      sqlserver-se: sqlserver-se
+      sqlserver-web: sqlserver-web
+  - field_name: mssql_version
+    required: true
+    type: string
+    details: |
+      The version for the MSSQL instance. Can be any version supported by the provider.
+  - field_name: storage_gb
+    required: true
+    type: number
+    details: Size of storage volume for service instance.
+    default: 20
+    constraints:
+      maximum: 4096
```

Comment:
nitpick: the maximum value depends on the storage type. I would delete the constraint